Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -919,6 +919,11 @@ theorem IsRegular.ord_pos {c : Cardinal} (H : c.IsRegular) : 0 < c.ord := by
 theorem isRegular_cof {o : Ordinal} (h : o.IsLimit) : IsRegular o.cof :=
   ⟨aleph0_le_cof.2 h, (cof_cof o).ge⟩
 
+/-- If `c` is a regular cardinal, then `c.ord.toType` has a least element. -/
+noncomputable def IsRegular.orderBotOrdToType
```

Comment:
I don't think we need this - if anything, we should instead have `IsRegular.ne_zero` instead so that we can simply write `c.orderBotToType H.ne_zero`.